src/opnsense/mvc/app/models/OPNsense/Backup/NextcloudSettings.xml

```diff
@@ -48,9 +48,9 @@
         </password_encryption>
         <backupdir type="TextField">
             <Required>Y</Required>
-            <mask>/[a-z0-9\-]+/i</mask>
+            <mask>/^[\w\-][\w\-\/]+[\w\-]$/</mask>
```

Comment:
I think this regex is wrong. I think you meant something like this:
```suggestion
            <mask>/^([\w%+\-]+\/)*[\w+%\-]+$/</mask>
```
This means the first part is entirely optional (allows names like 'a') and if it is in place, It allows names like 'aa'.

You can test regexes on rubular if you want. Here are some examples;
```
abc
a/b/%20+/aa/xaajsdkjfallsdjfas
a/b/%20+/aa/xaajsdkjfallsdjfas%20
a/b/%20+/aa/xaajsdkjfallsdjfas+
a//b
```
The last one does not match so the double slash is detected as well! Important things: % is used for encoding using hex like %20 for space. Also + is used for spaces so this characters should be allowed as well.